From: Andre Moreira Magalhaes <andre@endlessm.com>
Date: Mon, 17 Aug 2020 18:41:04 -0300
Subject: authPrompt: Properly get oVirt service name

Commit 809f820cd moved the definition of OVIRT_SERVICE_NAME to
'js/gdm/oVirt.js' but missed updating one occurrence. This
commit fixes it.

https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/1414

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/0043775

Bug-Ubuntu: https://bugs.launchpad.net/bugs/1886592
---
 js/gdm/authPrompt.js | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/js/gdm/authPrompt.js b/js/gdm/authPrompt.js
index e535d0f..a6ea6c8 100644
--- a/js/gdm/authPrompt.js
+++ b/js/gdm/authPrompt.js
@@ -6,10 +6,11 @@ const { Clutter, GObject, Pango, Shell, St } = imports.gi;
 const Animation = imports.ui.animation;
 const Batch = imports.gdm.batch;
 const GdmUtil = imports.gdm.util;
-const Util = imports.misc.util;
+const OVirt = imports.gdm.oVirt;
 const Params = imports.misc.params;
 const ShellEntry = imports.ui.shellEntry;
 const UserWidget = imports.ui.userWidget;
+const Util = imports.misc.util;
 
 var DEFAULT_BUTTON_WELL_ICON_SIZE = 16;
 var DEFAULT_BUTTON_WELL_ANIMATION_DELAY = 1000;
@@ -482,7 +483,7 @@ var AuthPrompt = GObject.registerClass({
             // The user is constant at the unlock screen, so it will immediately
             // respond to the request with the username
             beginRequestType = BeginRequestType.PROVIDE_USERNAME;
-        } else if (this._userVerifier.serviceIsForeground(GdmUtil.OVIRT_SERVICE_NAME) ||
+        } else if (this._userVerifier.serviceIsForeground(OVirt.SERVICE_NAME) ||
                    this._userVerifier.serviceIsForeground(GdmUtil.SMARTCARD_SERVICE_NAME)) {
             // We don't need to know the username if the user preempted the login screen
             // with a smartcard or with preauthenticated oVirt credentials
